easybuild/cray_external_modules_metadata-20.05.cfg

```diff
@@ -0,0 +1,122 @@
+# metadata useful to EasyBuild (hpcugent.github.io/easybuild) for modules provided by Cray on Cray systems
+# authors: Guilherme Peretti-Pezzi (CSCS), Petar Forai (IMP/IMBA), Kenneth Hoste (HPC-UGent), Eirini Koutsaniti (CSCS)
```

Comment:
If the file is automatically generated, I think it is good to mention it here.

Reply:
This was not automatically generated, because this time I had to check which variables were added to the new modules and will be recognized by easybuild. But in theory, after we drop the old versions of python, it should be only CUDA that needs to be updated, since we will have to include the version in the index of the entry.